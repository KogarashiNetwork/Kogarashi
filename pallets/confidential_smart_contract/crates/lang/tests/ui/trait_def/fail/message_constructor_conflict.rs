use ink_lang as ink;

#[ink::trait_definition]
pub trait TraitDefinition {
    #[ink(message, constructor)]
    fn message(&self);
}

fn main() {}
