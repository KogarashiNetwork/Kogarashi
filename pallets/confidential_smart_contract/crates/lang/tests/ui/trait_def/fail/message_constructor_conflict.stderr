error: encountered conflicting ink! attribute argument
 --> tests/ui/trait_def/fail/message_constructor_conflict.rs:5:20
  |
5 |     #[ink(message, constructor)]
  |                    ^^^^^^^^^^^
